docs/maintainers/README.md

```diff
@@ -28,6 +28,106 @@ The final step to preparing your language to accept contributions is to provide
 
 We recommend choosing a Concept that involves working with basic types, like strings and numbers. Having chosen a Concept, look for the Concept's description in this reference part of this repository. This file should have references to all tracks that have implemented the exercise. You can use these existing implementations as a starting point for your first concept exercise, which you can then tailor to your specific language.
 
+### Planning the track structure
```

Comment:
Maybe do:

```suggestion
### Visualizing the track structure
```

as the content seems to deal with the visualization aspect, not on the actual planning. Just a suggestion.

Reply:
I want to emphasize that it's a planning tool. For visualising the track structure of implemented exercises, this is probably not all that useful and you'd want to generate a graph from the `config.json` instead.

Perhaps `### Visualizing the planned track structure`?

Reply:
```suggestion
### Visualizing the planned track structure
```

Reply:
I like that!